lib/cadet/auth/error_handler.ex

```diff
@@ -2,13 +2,18 @@ defmodule Cadet.Auth.ErrorHandler do
   @moduledoc """
   Handles authentication errors
   """
+  use CadetWeb, :controller
```

Comment:
This is not a controller. If you want the `session_path` get it from `import CadetWeb.Router.Helpers, only: [session_path: 2]`